views/header.html

```diff
@@ -11,12 +11,29 @@
 <link href=/feeds/rss  rel=alternate type=application/rss+xml>
 <link href=/feeds/json rel=alternate type=application/json>
 
+{{ $title := "Code Golf" }}
+{{ with .Title }}{{ $title = (print . " | Code Golf") }}{{ end }}
+<title>{{ $title }}</title>
+<meta property="og:title" content="{{ $title }}">
+
 {{/* FIXME This won't change for dark themes :-( */}}
 <meta name=theme-color content=#343A40>
 <meta name=viewport    content="width=device-width">
-<meta name=description content="Code Golf is a game designed to let you show off your code-fu by solving problems in the least number of characters.">
+{{ $description := "Code Golf is a game designed to let you show off your code-fu by solving problems in the least number of characters." }}
+<meta name=description content="{{ $description }}">
+<meta property="og:description" content="{{ $description }}">
+<meta name="twitter:card" content="summary">
+
+{{ if .GolferInfo }}
+<meta property="og:image" content="https://avatars.githubusercontent.com/{{ .GolferInfo.Name }}?s=200">
+{{ else }}
+<meta property="og:image" content="https://code.golf/icon-180.png">
+{{ end }}
+<meta property="og:image:width" content="200">
+<meta property="og:image:height" content="200">
+<meta property="og:url" content="https://code.golf{{ with .Path }}{{ . }}{{ end }}">
 
-<title>{{ with .Title }}{{ . }} | {{ end }}Code Golf</title>
+{{ if .Beta }}<meta name=robots content=noindex>{{ end }}
```

Comment:
This looks like an artefact from a rebase, the beta site doesn't exist anymore.

Reply:
My bad, I'll clean this up :)